lib/statsd.js

```diff
@@ -9,6 +9,13 @@ var StatsD = require('hot-shots');
  */
 var DEFAULT_MAX_BATCH_SIZE = 1450;
 
+function objectAssign(target, source) {
+    Object.keys(source).forEach(function(keyName) {
```

Comment:
Perhaps it'd be worth checking if `Object.assign` exists, and if it does use that, otherwise fall back to the polyfill